### ISSUE TITLE
not work for drop dead

### ISSUE THREAD

**Issue by seophone1982** [ahmanwoods/Oculus-Defibrillator#3]
hello，I tried heartbeat patch，found it wont work for drop dead，cant found the second player who should be joined my room，can you fix it？

**Comment by ahmanwoods** [comment]
Did you put the correct AppID in the script?

**Comment by seophone1982** [comment]
> Did you put the correct AppID in the script?

yes，I do use the correct AppID 1091604990907791
by the way I am using quest2 + link cable

**Comment by ahmanwoods** [comment]
May I ask why you need this script for Quest 2 + Link? The game should run natively through Oculus.

**Comment by seophone1982** [comment]
> May I ask why you need this script for Quest 2 + Link? The game should run natively through Oculus. 

for better graph

**Comment by ahmanwoods** [comment]
Right, I understand why you want to run the game on the PC, but you can install the game and run it natively through the desktop oculus client, no need for Revive.
